feature-libs/product-configurator/rulebased/occ/variant/variant-configurator-occ.adapter.ts

```diff
@@ -324,6 +324,12 @@ export class VariantConfiguratorOccAdapter
     );
     //no need to work with a converter here, as Configurator.Variant is a projection of the OCC
     //variant representation
-    return this.http.get<Configurator.Variant[]>(url);
+    return this.http.get<Configurator.Variant[]>(url, { context: this.prepareSendUserContext() });
+  }
+
+  prepareSendUserContext(): HttpContext {
+    return new HttpContext().set(OCC_HTTP_TOKEN, {
```

Comment:
And please also add a JS doc that states that the actual calls to the commerce backend will only be changed if the ASM setting 
```
    userIdHttpHeader:{
      enable:true
    },
```
is active